src/Microsoft.OData.Client/MaterializeFromAtom.cs

```diff
@@ -104,7 +104,8 @@ internal class MaterializeAtom : IDisposable, IEnumerable, IEnumerator
         /// <param name="elementType">result type.</param>
         /// <param name="format">The format of the response being materialized from.</param>
         /// <param name="materializerCache">Cache used to store temporary metadata used for materialization of OData items.</param>
-        internal MaterializeAtom(ResponseInfo responseInfo, IEnumerable<ODataResource> entries, Type elementType, ODataFormat format, MaterializerCache materializerCache)
+        /// <param name="includeLinks">Whether to include navigation properties when materializing an entry.</param>
+        internal MaterializeAtom(ResponseInfo responseInfo, IEnumerable<ODataResource> entries, Type elementType, ODataFormat format, MaterializerCache materializerCache, bool includeLinks = true)
         {

```

Comment:
here too? should the default `includeLinks` value be true or false?

Reply:
true. As explained above